core/src/main/java/org/apache/iceberg/AllManifestsTable.java

```diff
@@ -44,13 +44,17 @@
  */
 public class AllManifestsTable extends BaseMetadataTable {
   private static final Schema MANIFEST_FILE_SCHEMA = new Schema(
+      Types.NestedField.required(14, "content", Types.IntegerType.get()),
```

Comment:
Using fresh IDs but adding columns in places where they make sense instead of adding to the end.